packages/dredd/test/fixtures/json-schema-draft-7-boolean.apib

```diff
@@ -0,0 +1,21 @@
+FORMAT: 1A
+
+# Machines API
+
+# Group Machines
+
+# Machines collection [/machines/{id}]
+    + Parameters
+        + id (number, `1`)
```

Comment:
This is a much older parameter syntax in API Blueprint from before it was aligned with MSON. If I recall correctly, this older synatx is undocumented in API Blueprint. Wondering where you found it? (perhaps in other fixtures?)

I'd recommend moving to the newer syntax:

```suggestion
        + id: 1 (number)
```